server/templates.go

```diff
@@ -89,37 +77,32 @@ func loadWebConfig(c webConfig) (http.Handler, http.Handler, *templates, error)
 	if c.issuer == "" {
 		c.issuer = "dex"
 	}
-	if c.dir == "" {
-		c.dir = "./web"
```

Comment:
So it used to default to `"./web"`, and now it would crash if both `Dir == ""` and `WebFS == nil`?

Reply:
Good call. It shouldn't assume one of `dir` or `webFS` is set. Will make it default to `./web` if both not set.

Reply:
I think this should be resolved as high in the stack as possible and only an FS instance should be injected:

- if dir is set use it
- if not, fallback to ./web

Eventually, we should embed assets anyway and use that as a default instead of ./web